packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,156 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * import { PaginationFieldMessageEmbed } from '@sapphire/discord.js-utilities';
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('Test pager field')
+ *    .setTemplate({ embed })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
+ *    .setItemsPerPage(2)
+ *    .make()
+ *    .run(message);
+ * ```
+ */
+export class PaginationFieldMessageEmbed<Element> extends PaginatedMessage {
+	private embedTemplate: MessageEmbed = new MessageEmbed();
+	private totalPages = 0;
+	private items: Element[] = [];
+	private itemsPerPage = 10;
+	private fieldTitle!: string;
+
+	/**
+	 * Items
+	 *
+	 * Array of items for pagination.
+	 * @param items The pages to set
+	 */
+	public setItems(items: Element[]) {
+		this.items = items;
+		return this;
+	}
+
+	/**
+	 * Title
+	 *
+	 * Title of the embed field that will be used to paginate the items.
+	 * @param title Field title
+	 */
+	public setTitleField(title: string) {
+		this.fieldTitle = title;
+		return this;
+	}
+
+	/**
+	 * items per page
+	 * Number of items to show per page.
+	 * @param itemsPerPage Number o items
+	 */
+	public setItemsPerPage(itemsPerPage: number) {
+		this.itemsPerPage = itemsPerPage;
+		return this;
+	}
+
+	/**
+	 * Embed template
+	 *
+	 * Template to be used to display the pagination of the articles. This template can be configured by the `new MessageEmbed()` class or directly by means of an object with embed options.
+	 *
+	 * @example
+	 * ```typescript
+	 * import { PaginationFieldMessageEmbed } from '@sapphire/discord.js-utilities';
+	 * import { MessageEmbed } from 'discord.js';
+	 *
+	 * // Option 1:
+	 * new PaginationFieldManager().setTemplate(new MessageEmbed().setTitle('Test pager embed')).make().run(message);
+	 * // Option 2:
+	 * new PaginationFieldManager().setTemplate({ title: 'Test pager embed' }).make().run(message);
+	 * ```
+	 * @param template MessageEmbed
+	 */
```

Comment:
```suggestion
	/**
	 * Sets the template to be used for each of the pages. This template can be configured by the `new MessageEmbed()` class, or by passing an object of {@link MessageEmbedOptions}.
	 *	 
	 * @param template The template to set
	 *
	 * @example
	 * ```typescript
	 * import { PaginationFieldMessageEmbed } from '@sapphire/discord.js-utilities';
	 * import { MessageEmbed } from 'discord.js';
	 *
	 * new PaginationFieldMessageEmbed().setTemplate(new MessageEmbed().setTitle('Test pager embed')).make().run(message);
	 * ```
	 * 
	 * @example
	 * ```typescript
	 * import { PaginationFieldMessageEmbed } from '@sapphire/discord.js-utilities';
	 * import { MessageEmbed } from 'discord.js';
	 *
	 * new PaginationFieldMessageEmbed().setTemplate({ title: 'Test pager embed' }).make().run(message);
	 * ```
	 */
```